src/pages/Controllers/Controllers.js

```diff
@@ -386,7 +386,7 @@ export default function Controllers() {
       <Header>
         <div className="controllers--count">
           {controllerCount} controllers,{" "}
-          <a href="/models">{modelCount} models</a>
+          <a href="/dashboard/models">{modelCount} models</a>
```

Comment:
This is actually only the correct root path when deployed via Juju directly and not in any other environment. Thanks for finding this bug but I THINK the fix is actually to use the `Link` component from `react-router-dom` so that it resolves correctly using the defined base path.

I'm actually working in this file right now so I can include this fix as a driveby.....or if you'd like you can update this PR :)

(untested)
```diff
diff --git a/src/pages/Controllers/Controllers.js b/src/pages/Controllers/Controllers.js
index 6a06853..3cd2dcb 100644
--- a/src/pages/Controllers/Controllers.js
+++ b/src/pages/Controllers/Controllers.js
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import { useDispatch, useSelector, useStore } from "react-redux";
+import { Link } from "react-router-dom";
 import cloneDeep from "clone-deep";
 import classNames from "classnames";

@@ -386,7 +387,7 @@ export default function Controllers() {
       <Header>
         <div className="controllers--count">
           {controllerCount} controllers,{" "}
-          <a href="/models">{modelCount} models</a>
+          <Link to="/models">{modelCount} models</Link>
         </div>
       </Header>
       <div className="l-content controllers">
```